src/component/strategies/ShowStrategy.tsx

```diff
@@ -0,0 +1,85 @@
+import {
```

Comment:
We need to land a naming convention on these views. Right now we have too many different ones. I would like us to either use StrategyView or something similar for now.